lib/tool_shed/test/base/test_db_util.py

```diff
@@ -90,15 +91,13 @@ def get_installed_repository_by_id(repository_id):
     )
 
 
-def get_installed_repository_by_name_owner(repository_name, owner, return_multiple=False):
-    query = (
-        install_session()
-        .query(galaxy.model.tool_shed_install.ToolShedRepository)
-        .filter(
-            and_(
-                galaxy.model.tool_shed_install.ToolShedRepository.table.c.name == repository_name,
-                galaxy.model.tool_shed_install.ToolShedRepository.table.c.owner == owner,
-            )
+def get_installed_repository_by_name_owner(repository_name, owner, return_multiple=False, session=None):
+    if session is None:
+        session = install_session()
+    query = session.query(galaxy.model.tool_shed_install.ToolShedRepository).filter(
```

Comment:
convert to `select` construct? Then, in the return clause, just `session.scalars(stmt).all()` for multiple and
`session.scalars(stmt.limit(1)).first()` for one (note the limit clause).